src/settings/utils.js

```diff
@@ -4,7 +4,7 @@ import { FormattedMessage } from 'react-intl';
 const NAME_REGEXP = new RegExp(/[^a-zA-Z\d]/);
 
 export const validatePrefixSuffixName = ({ name }) => ({
-  name: name?.match(NAME_REGEXP)
+  name: name?.match(NAME_REGEXP) || name?.length > 7
```

Comment:
@AndreiShumski2 if you use regexp maybe it's better to check length there as well